lib/helpers/contacts/Recipients.php

```diff
@@ -0,0 +1,102 @@
+<?php
+/**
+ * This helper builds a html form and provides a submission endpoint for the form that makes a /contactdb/recipients API call.
+ *
+ * PHP version 5.6, 7
+ *
+ * @author    Kraig Hufstedler <kraigory@gmail.com>
+ * @copyright 2017 SendGrid
+ * @license   https://opensource.org/licenses/MIT The MIT License
+ * @version   GIT: <git_id>
+ * @link      http://packagist.org/packages/sendgrid/sendgrid
+ */
+namespace SendGrid;
+
+class RecipientForm
+{
+    public $html;
```

Comment:
Why are the properties public?, we're breaking OOP encapsulation doing this.

Reply:
Will change.